index.js

```diff
@@ -22,7 +22,7 @@ function stubMethod(service, method, replacement) {
   if (!isStubbed(service)) stubService(service);
   if (!replacement) return sinon.stub(getService(service).prototype, method);
 
-  return sinon.stub(getService(service).prototype, method, function(params, callback) {
+  return sinon.stub(getService(service).prototype, method).callsFake(function(params, callback) {
```

Comment:
Upgrading to 2.x prints out a warning when running tests like this:

```
sinon.stub(obj, 'meth', fn) is deprecated and will be removed from the public API in a future version of sinon.
 Use stub(obj, 'meth').callsFake(fn).
 Codemod available at https://github.com/hurrymaplelad/sinon-codemod
```

Testing with 

```
"@mapbox/mock-aws-sdk-js": "mapbox/mock-aws-sdk-js#d307e61e620bf3c14cab1fd177110dce7f983af1",
```

installed on libraries where we're using this module seem to keep testing as expected. I can do a few more runs across different libraries just to confirm, though.